library/sinks/ChildProcess.test.ts

```diff
@@ -80,4 +92,28 @@ t.test("it works", async (t) => {
       "Aikido runtime has blocked a Shell injection: child_process.execSync(...) originating from body.file.matches"
     );
   });
+
+  runWithContext(unsafeContext, () => {
+    throws(
+      () =>
+        spawn(
+          "ls `echo .`",
+          [],
+          { shell: true },
+          (err, stdout, stderr) => {}
+        ).unref(),
+      "Aikido runtime has blocked a Shell injection: child_process.spawn(...) originating from body.file.matches"
+    );
+
+    throws(
+      () =>
+        spawnSync(
+          "ls `echo .`",
+          [],
+          { shell: true },
```

Comment:
Maybe set this to `"/bin/sh"` or something like that? To cover that path?

Including when it's set to `false`?

Reply:
Should be resolved now, I have included several test cases for when `shell` is a string, false, etc.